### PR TITLE
Polishes text premine page

### PR DESCRIPTION
**Pull request by s-ben**
This commit polishes text on the new premine page, which was a bit rough. Notably, the page reused text from an old announcement, which was in present tense. Have reworded to use past tense, wordsmithed for flow, and improved formatting.

### CODE REVIEW

File: docs/advanced/premine.md [hunk 1]
```diff
@@ -2,29 +2,29 @@
...
 
-Rather than allocating the entire premine to the bring-up costs, the remaining half of the premine was spread evenly across a list of airdrop participants. Sign-up for the airdrop opened with a public announcement on December 15th, 2015 and closed on January 18th, 2016. Not everybody who signed up was selected to participate in the airdrop - Decred is fundamentally about technological progress, so the airdrop targeted individuals that have made contributions to advance technology in its various forms. There was also a large number of fraudulent sign-ups which needed to be carefully identified and dealt with.
+Sign-up for the airdrop opened with a public announcement on December 15th, 2015 and closed on January 18th, 2016. Not all participants who signed up were selected to participate in the airdrop - Decred is fundamentally about technological progress, so the airdrop targeted individuals that have made contributions to advanced technology in its various forms. There were also a large number of fraudulent sign-ups, which were carefully identified and dealt with.
```
Comment (jholdstock) [inline docs/advanced/premine.md]:
```suggestion
Sign-up for the airdrop opened with a public announcement on December 15th, 2015 and closed on January 18th, 2016. Not all participants who signed up were selected to participate in the airdrop - Decred is fundamentally about technological progress, so the airdrop targeted individuals that have made contributions to advanced technology in its various forms. There were also a large number of fraudulent sign-ups, which were carefully identified and dealt with.
```

I believed `advanced` here should either be `advance` or `advancing`

File: docs/advanced/premine.md [hunk 2]
```diff
@@ -2,29 +2,29 @@
...
 
-> The airdrop concluded with awarding 282.63795424 DCR to 2,972 participants.
+> When the airdrop concluded, 282.63795424 DCR was awareded to 2,972 participants.
```
Comment (jholdstock) [inline docs/advanced/premine.md]:
```suggestion
> When the airdrop concluded, 282.63795424 DCR was awarded to 2,972 participants.
```

File: mkdocs.yml [hunk 3]
```diff
@@ -59,7 +59,7 @@ nav:
   - 'Obtaining DCR': 'getting-started/obtaining-dcr.md'
   - 'Using the Block Explorer': 'getting-started/using-the-block-explorer.md'
 - Governance:
-  - 'Introduction to Decred Governance': 'governance/introduction-to-decred-governance.md'
+  - 'Overview': 'governance/introduction-to-decred-governance.md'
```
Comment (jholdstock) [inline mkdocs.yml]: These changes don't belong in this PR. Probably left over from your #914 branch

**Comment by s-ben** [comment]
@jholdstock good suggestions, implemented. Should be good to go.

File: docs/advanced/premine.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (s-ben) [reply docs/advanced/premine.md]: Should be fixed in the last commit (was flagged by @jholdstock  in his review).
